src/status_im/contexts/communities/actions/accounts_selection/view.cljs

```diff
@@ -16,6 +17,7 @@
   []
   (let [{id :community-id} (rf/sub [:get-screen-params])
         {:keys [name color images joined]} (rf/sub [:communities/community id])
+        {:keys [has-permissions?]} (rf/sub [:community/token-permissions id])
```

Comment:
if you are pulling a single key, I think it's best to just create a new subscription